### PR TITLE
Updated landing page tile padding, margin for responsive design

### PR DESCRIPTION
**Pull request by yuinchien**

### CODE REVIEW

File: app/images/thumbnails/thumb_pwa.svg [hunk 1]
```diff
@@ -1 +1,22 @@
...
+            </g>
+            <g id="Group-9" transform="translate(200.000000, 0.000000)">
+                <g id="Group-17">
+                    <rect id="Rectangle" stroke="#FFFFFF" stroke-width="24" opacity="0.49562872" x="12" y="12" width="160" height="160" rx="80"></rect>
+                    <rect id="Rectangle-Copy-36" x="26.2280702" y="26.2280702" width="132.25" height="132.25" rx="66.125"></rect>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (keanulee) [inline app/images/thumbnails/thumb_pwa.svg]: To be run through minifier (or remind @keanulee to do this after this PR).

File: app/index.html [hunk 2]
```diff
@@ -26,17 +26,21 @@ <h1 class="hero-title">Polymer Project</h1>
...
       display: inline-block;
-      width: calc(50% - 50px);
-      margin-bottom: 20px;
+      width: calc(50% - 60px);
+      margin-bottom: 40px;
       padding: 20px;
       background: rgba(0,0,0,0.6);
       background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, rgba(0,0,0,0.4) 0%, rgba(0,0,0,0.6) 100%);
-      color: #000;
+      color: rgba(0,0,0,.87);
```
Comment (keanulee) [inline app/index.html]: Note: alpha text color here is fine - limited use and required for effect with gradient background.

File: app/index.html [hunk 3]
```diff
@@ -116,97 +122,101 @@ <h1 class="hero-title">Polymer Project</h1>
...
+      }
     }
   </style>
 
   <section>
     <div class="wrapper">
       <div>
-        <a class="product-grid-item left" href="https://lit-element.polymer-project.org" style="background: #00A3E6; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #3FC7FF 0%, #00A3E6 100%)">
-          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_litElement.svg); padding-top: 35%"></div>
+        <a class="onload-fadein product-grid-item left" href="https://lit-element.polymer-project.org" style="-webkit-animation-delay: .15s; background: #00A3E6; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #3FC7FF 0%, #00A3E6 100%)">
```
Comment (keanulee) [inline app/index.html]: Unprefix `animation-delay`, here and below

File: app/index.html [hunk 4]
```diff
@@ -116,97 +122,101 @@ <h1 class="hero-title">Polymer Project</h1>
...
-          <h2>About the Polymer Project</h2>
-          <div>
-            <p>As front-end engineers in the Chrome team, our mission is to make the web better.</p>
-
-            <p>We work on libraries and tools to help developers unlock the web’s full potential, taking advantage of cutting-edge features like Web Components, Service Workers and HTTP/2.</p>
-              
-            <p>We experiment with new patterns for building faster, smaller web applications.</p>
-              
-            <p>We advocate for standards, helping ensure that web developers have a strong voice in the process.</p>
+      <div class="wrapper">
```
Comment (keanulee) [inline app/index.html]: 2 space indentation (no diff expected below this point)

**Comment by yuinchien** [comment]
@keanulee i've updated the code following your comment, please review again, thanks!

**Comment by keanulee** [comment]
Thanks @yuinchien

File: app/index.html [hunk 5]
```diff
@@ -116,48 +122,52 @@ <h1 class="hero-title">Polymer Project</h1>
...
+        <a class="onload-fadein product-grid-item left" href="https://github.com/material-components/material-components-web-components" style="animation-delay: .25s; background: #00D67C; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #20FFA7 0%, #00D67C 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_material.svg); padding-top: 46%; margin: 10px 15%;"></div>
+          <h2>Material Web Components</h2>
+          <p>High-fidelity realizations of Google's Material Design spec that work anywhere on the web.</p>
+          <span class="status">Prerelease</span>
+        </a>
+        <a class="onload-fadein product-grid-item right" href="https://lit-html.polymer-project.org" style="animation-delay: .3s; background: #FF2F2F; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #FA5A5A 0%, #FF2F2F 100%)">
```
Comment (frankiefu) [inline app/index.html]: @yuinchien Just curious if the change in orders ("Material Web Components" before "lit-html") is intentional.

**Comment by yuinchien** [comment]
@frankiefu i tweaked the order of the tiles to create a more balanced layout, but it seems to break for certain window sizes, let me improve on this.

**Pull request merged**